anisoap/representations/radial_basis.py

```diff
@@ -272,3 +452,36 @@ def orthonormalize_basis(self, features: TensorMap):
                 )
 
         return features
+
+    def get_basis(self, rs):
+        from matplotlib import pyplot as plt
+
+        all_gs = np.empty(shape=(len(rs), 1))
```

Comment:
i don't really understand this and line 486 but it seems to produce the right basis functions so we can let it be.

Reply:
The np.empty? That's to create an empty list we can concatenate onto later with the right shape. There are other options for how to do it, this is just the one I chose.

Reply:
You good with this?